Comp2023-Imported/src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -29,17 +28,12 @@ public class Vision extends SubsystemBase
   public MedianFilter           m_tvfilter      = new MedianFilter(5); // median filter v values to remove outliers (5 sample)
 
   // Declare module variables
-  private double                m_distance1     = LLConsts.kDistance1;   // x position in inches for first reference point
-  private double                m_vertOffset1   = LLConsts.kVertOffset1; // y reading in degrees for first reference point
-  private double                m_distance2     = LLConsts.kDistance2;   // x position in inches for second reference point
-  private double                m_vertOffset2   = LLConsts.kVertOffset2; // y reading in degrees for second reference point
   private double                m_slope;   // Linear regressions slope from calibration

```

Comment:
slope and offset are also part of the festus limelight driving calibration--so shouldn't be needed.